services/rest/src/getDealStatus.ts

```diff
@@ -0,0 +1,64 @@
+import { APIGatewayProxyEventBase, APIGatewayProxyResult } from 'aws-lambda';
+import middy from '@middy/core';
+import cors from '@middy/http-cors';
+
+import Appsync from 'aws-appsync';
+import gql from 'graphql-tag';
+import { AuthContext } from './authorizer';
+
+require('cross-fetch/polyfill');
+
+const graphqlClient = new Appsync({
+  url: process.env.APPSYNC_URL,
+  region: process.env.AWS_REGION,
+  auth: {
+    type: 'AWS_IAM',
+    credentials: {
+      // is there a way to get these from the lambda role
```

Comment:
This piece I am not hundred percent on.  Will these env vars get populated or maybe there is a way to set this auth to use the default lambda execution role? That's what it should be because the permissions in serverless.yml are set for that.